packages/office-ui-fabric-react/src/components/Check/Check.scss

```diff
@@ -9,74 +9,73 @@
 
 $checkBoxHeight: 24px;
 
-:global {
-  .ms-Check {
-    line-height: 1;
-    width: $checkBoxHeight;
-    height: $checkBoxHeight;
-    vertical-align: top;
-    position: relative;
-    @include user-select(none);
+.root {
+  line-height: 1;
+  width: $checkBoxHeight;
+  height: $checkBoxHeight;
+  vertical-align: top;
+  position: relative;
+  @include user-select(none);
 
-    &.is-checked {
-      .ms-Check-background {
-        &:before {
-          color: $ms-color-themePrimary;
-        }
-
-        &:after {
-          color: $ms-color-white;
-        }
+  &.isChecked {
```

Comment:
Rather than nesting, we can use `rootIsChecked`. Or do we want to preserve the existing nesting and save flattening this out for later?